pyfstat/core.py

```diff
@@ -626,6 +632,10 @@ def init_computefstatistic(self):
             ] = self.injectSqrtSX
         else:
             FstatOAs.injectSqrtSX = lalpulsar.FstatOptionalArgsDefaults.injectSqrtSX
+        if hasattr(self, "randSeed") and self.randSeed is not None:
+            FstatOAs.randSeed = self.randSeed
+        else:
+            FstatOAs.randSeed = lalpulsar.FstatOptionalArgsDefaults.randSeed
```

Comment:
```suggestion
        FstatOAs.randSeed = (
              getattr(self, "randSeed", None) or lalpulsar.FstatOptionalArgsDefaults.randSeed
       )
```


I agree with this. I tried to simplify it, but I *don't* want to prevent 0 from being used as a proper seed.
For the sake of it, I'll leave it as a suggestion.

Q: Is there any reason not to use 0 as a seed in a random number generator?

Reply:
Actually that's a good point: I just checked and the corresponding CFSv2 help is `pecify random-number seed for reproducible noise (0 means use /dev/urandom for seeding).`, and indeed that's what the low-level code does: https://lscsoft.docs.ligo.org/lalsuite/lalpulsar/_generate_pulsar_signal_8c_source.html#l01002

So there actually wouldn't be a problem if we interpret a user-given `0` as "give me the default" at the same level as a `None`, and our documentation should be explicit about that too.

Reply:
See latest commit.

Reply:
Added a bit more verbosity to the docstring. Cool!